src/commands/index.ts

```diff
@@ -534,6 +539,77 @@ export class VSCodeCommands implements Disposable {
           this.dbtTerminal.logLine(`Error=${e}`);
         }
       }),
+      commands.registerCommand("dbtPowerUser.createSqlFile", async () => {
+        try {
+          const project =
+            await this.queryManifestService.getOrPickProjectFromWorkspace();
+          if (!project) {
+            window.showErrorMessage("No dbt project selected.");
+            return;
+          }
+
+          const uri = Uri.parse(
+            `${project.projectRoot}/poweruser-${getFormattedDateTime()}.sql`,
+          ).with({ scheme: "untitled" });
+          const annotationDecoration: TextEditorDecorationType =
+            window.createTextEditorDecorationType({
+              rangeBehavior: DecorationRangeBehavior.OpenOpen,
+            });
+
+          const contentText =
+            "Enter your query here and execute it just like any dbt model file. This file is unsaved, you can either save it to your project or save it as a bookmark for later usage or share it with your team members.";
```

Comment:
we probably should make this a sql comment.

Reply:
Initially we had it like that, but it adds to the saved query history. so we want to show as just one time text for every new editor